airbyte-integrations/connectors/source-faker/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_faker ./source_faker
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.2.1
+LABEL io.airbyte.version=0.3.0
```

Comment:
Shall we realease a major version as the config backward incompatible with 0.2.1 according to `acceptance-test-config.yml`?